src/engine/UIManager.cxx

```diff
@@ -129,6 +130,7 @@ void UIManager::init()
 
       if (!element["Type"].is_null())
       {
+        std::string elmType = element.value("Type", "");
```

Comment:
We already have a line of code getting the type a few lines down:
`auto uiElementType = element.value("Type", "");`

Reply:
ok